collectors/build/jenkins-audit/src/main/java/com/capitalone/dashboard/util/AuditDataCSVFileCreator.java

```diff
@@ -0,0 +1,275 @@
+package com.capitalone.dashboard.util;
+
+import com.capitalone.dashboard.model.AuditResult;
+import com.capitalone.dashboard.model.AuditType;
+import com.capitalone.dashboard.model.Cmdb;
+import com.capitalone.dashboard.model.Dashboard;
+import com.capitalone.dashboard.repository.CmdbRepository;
+import com.capitalone.dashboard.repository.DashboardRepository;
+import com.capitalone.dashboard.response.AuditReviewResponse;
+import com.capitalone.dashboard.response.DashboardReviewResponse;
+import com.capitalone.dashboard.service.DashboardAuditService;
+import com.capitalone.dashboard.status.DashboardAuditStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+/**
+ * <h1>AuditDataCSVFileCreator</h1>
+ * This class helps to create csv data from dashboard audit statuses
+ * <p>
+ * How to use?
+ * 1. Add @Component tag
+ * 2. Assign values - BEGIN_DATE, END_DATE, CSV_OUTPUT_FILE_PATH
+ * 3. Execute Application
+ *
+ * @since 09/28/2018
+ */
+@SuppressWarnings("PMD")
+public class AuditDataCSVFileCreator {
+
+    // BEGIN_DATE, END_DATE,CSV_OUTPUT_FILE_PATH values finalized by CSV file creator
+    private static final long BEGIN_DATE = 0;
+    private static final long END_DATE = 0;
+    private static final String CSV_OUTPUT_FILE_PATH = "";
+
+    private static final Set<AuditType> AUDIT_TYPE = new HashSet<>();
+    private static final String[] CSV_HEADER = {"LOB", "ASV", "BAP",
+            "ASV_OWNER", "SERV_OWNER", "AUDIT_TYPE", "COLLECTION_STATUS", "AUDIT_STATUS", "URL", "ERROR"};
+    private static final String[] AUDIT_TYPES = {"LIBRARY_POLICY", "CODE_REVIEW", "TEST_RESULT", "STATIC_SECURITY_ANALYSIS",
+            "PERF_TEST", "CODE_QUALITY", "BUILD_REVIEW"};
+    private static final String[] CODE_REVIEW_FAIL_STATUES = {"PEER_REVIEW_LGTM_ERROR", "PEER_REVIEW_LGTM_SELF_APPROVAL", "PEER_REVIEW_GHR_SELF_APPROVAL",
```

Comment:
please create an enum for all the Statuses. You can refer existing api-audit status classes Ex: CodeQualityAuditStatus

Reply:
Done